tests/test_core.py

```diff
@@ -868,10 +868,10 @@ def test_emmalloc(self, *args):
     self.set_setting('MALLOC', 'none')
     self.emcc_args += ['-fno-builtin'] + list(args)
 
-    self.do_run(open(path_from_root('system', 'lib', 'emmalloc.cpp')).read() +
-                open(path_from_root('system', 'lib', 'sbrk.c')).read() +
-                open(test_file('core/test_emmalloc.cpp')).read(),
-                open(test_file('core/test_emmalloc.out')).read())
+    self.do_run(read_file(path_from_root('system', 'lib', 'emmalloc.cpp')) +
```

Comment:
Can we use pathlib in `path_from_root` too so we can write like
```py
path_from_root('system/lib/emmalloc.cpp')
```
? (not necessarily in this PR)

Reply:
Thanks!  Done.